client/dashboard/sites/plans.ts

```diff
@@ -1,6 +1,14 @@
 import { DotcomPlans } from '../data/constants';
 import type { Site } from '../data/types';
 
+export const isSitePlanOneOf = ( site: Site, plans: DotcomPlans[] ) => {
```

Comment:
I'm not a massive fan of creating a wrapper for `.includes()` 🤷 

```ts
[
	DotcomPlans.ECOMMERCE_TRIAL_MONTHLY,
	DotcomPlans.HOSTING_TRIAL_MONTHLY,
	DotcomPlans.MIGRATION_TRIAL_MONTHLY,
].includes( site.plan ?? '' )
```

doesn't seem that bad IMO

Reply:
Fair enough. 😄